packages/frontend/src/contexts/game/actions/index.js

```diff
@@ -0,0 +1,11 @@
+import types from './types';
+
+export const addTiles = (photos) => ({
+  type: types.ADD_TILES,
+  payload: { photos },
+});
+
+export const flipTile = (tile) => ({
+  type: types.FLIP_TILE,
+  payload: { tile },
+});
```

Comment:
I read about toStrictEquals because I saw you used it in the other action tests. I basically type checks the object so that it can't be something other than an object literal. Got it.